### DISCUSSION TITLE
Create multiple Assistants

### DISCUSSION THREAD

**Discussion by mjamalm** [janhq/jan#1733] [Feature Ideas]
I'm thinking about creating multiple assistants or agents that I can pick,  for each agent there will be custom instructions so to make it easier for me to pick the agents for different models.

**Comment by TomLucidor** [comment]
Is it like AutoGen or CAMEL or BabyAGI or MetaGPT? Also thinking how SillyTavern behaves similarly without agent features

**Comment by sixlive** [comment]
This would be a massive feature add. I already drop in and customize the base Jan assistant with my own prompt, kind of a pain to manage when it gets wiped out during updates.

**Comment by sreejinsreenivasan** [comment]
I would like if this have agents for specific things, eg; Research.

**Comment by Unlok-Simon** [comment]
I am doing this (sort of) currently by creating a thread with custom instructions, choosing a model, giving it a descriptive name, and cleaning the thread each time I want to do something new with it. Would be great to have actual assistants you simply start a new thread with so I don't lose previous chats  but it is working for my regular tasks.

**Comment by tenten71** [comment]
I'm not sure if this is the same as LM Studio's "saved presets" function (I like Jan better though). Something like that might be nice for this. Otherwise i would do like Unlock-Simon.
